go-fuzz/master.go

```diff
@@ -27,6 +33,8 @@ type Master struct {
 	statExecs     uint64
 	statRestarts  uint64
 	coverFullness int
+
+	writerSet *writerset.WriterSet
```

Comment:
please rename this var to something more descriptive, e.g. "statWriters" or "statWriterSet"